lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -524,6 +527,26 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
             , expectErrorMessage errMsg403NotDelegating
             ]
 
+    it "STAKE_POOLS_QUIT_03 - Can quit with rewards"
+        $ \ctx -> runResourceT $ do
+        (w, _) <- rewardWallet ctx
```

Comment:
why to use MIR wallet here??? It cannot participate in joining/quitting, right? What about using just `fixtureWallet` ie. normal Shelley wallet?